.circleci/config.yml

```diff
@@ -64,7 +64,7 @@ commands:
           key: dependency-cache-{{ checksum "package.json" }}
       - run:
           name: Installing dependencies
-          command: node -v && npm install --legacy-peer-deps
+          command: node -v && npm install
```

Comment:
Do we want to use `npm ci`? It fails when `package-lock.json` changes would be made.

Reply:
@mhuynh5757 I'm not sure what you mean. It is working on CircleCI right now, with the `package-lock.json` changes.